### PR TITLE
simplify code contributions by automating the dev setup.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Changes:

Hi. I work for Gitpod(an online IDE which is free for Open Source) and we are currently on a mission to simplify contributors onboarding experience for cool and popular Open Source projects. 

This Pr adds Gitpod config to the repo via which, anyone would be able to launch a workspace where it will automatically:

- clone the `p5.js` repo.
- install all the dependencies i.e via running `npm ci`.
- run `npm run grunt` and `npm run dev` in separate terminals.

All of this can be helpful for newcomers/beginners i.e the can start in just a single click without having to set anything up.

You can give it a try on my fork of the repo via the following link:

https://gitpod.io/#https://github.com/nisarhassan12/p5.js

 Screenshots of the change:

image

#### PR Checklist

- [x] `npm run lint` passes
- [ ] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by nisarhassan12** [comment]
@all-contributors please add @nisarhassan12 for simplify code contributions.

**Comment by lmccart** [comment]
@nisarhassan12 this looks cool, but our [contribution process](https://github.com/processing/p5.js/tree/main/contributor_docs#github-issue-flow) asks that before a pull request is made, an issue is opened for discussion. could you please open an issue so we can discuss this addition? thanks!

**Comment by nisarhassan12** [comment]
@lmccart Sorry for not following the contributions process as specified. I have opened this issue https://github.com/processing/p5.js/issues/4882 for the discussion.

File: tasks/test/mocha-chrome.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = function(grunt) {
     // Launch Chrome in headless mode
     const browser = await puppeteer.launch({
       headless: true,
-      args: ['--no-sandbox']
+      args: ['--no-sandbox', '--disable-setuid-sandbox']
```
Comment (nisarhassan12) [inline tasks/test/mocha-chrome.js:19]: In order for the test suite to work this change has to be merged.

Comment (limzykenneth) [reply tasks/test/mocha-chrome.js:19]: It's probably fine to add but I would like to test this on a Linux machine before commiting it.

Comment (outofambit) [reply tasks/test/mocha-chrome.js:19]: for anyone coming here later, this change was added to `main` in https://github.com/processing/p5.js/pull/4959/commits/e9f04db7880a797afae3d6e13862f0ddd2fef592

**Pull request closed without merging**